com.github.gabutakut.gabutdm.yml

```diff
@@ -0,0 +1,48 @@
+app-id: com.github.gabutakut.gabutdm
+runtime: org.gnome.Platform
+runtime-version: '41'
+sdk: org.gnome.Sdk
+command: com.github.gabutakut.gabutdm
+finish-args:
+  - '--share=network'
+  - '--talk-name=com.canonical.Unity.LauncherEntry'
+  - '--share=ipc'
+  - '--socket=fallback-x11'
+  - '--socket=wayland'
+  - '--socket=x11'
+  - '--filesystem=xdg-download'
+cleanup:
+  - '*.a'
+  - '*.la'
+  - '*.o'
+  - '/include'
+modules:
+  - name: libcanberra
+    buildsystem: autotools
+    sources:
+      - type: git
+        url: git://git.0pointer.net/libcanberra.git
+        tag: master
```

Comment:
`master` isn't a tag. It's a branch. You should have a specific commit. Or better use `shared-modules` as it has all that is needed to run it.

Reply:
this wasn't resolved.